app/services/metrics/weekly/review_turnaround/per_user_project.rb

```diff
@@ -0,0 +1,52 @@
+module Metrics
+  module Weekly
+    module ReviewTurnaround
+      class PerUserProject < BaseService
+        def call
+          process
+        end
+
+        private
+
+        def process
+          daily_metrics.find_each do |daily_metric|
+            weekly_metric = weekly_of(daily_metric.ownable)
+            if weekly_metric.present?
+              next weekly_metric.update!(
```

Comment:
code like this looks weird to me
```ruby
if something
  next update
end

create
```

don't you think it'd be nicer a create or update? Is this codebase using Rails 6? If yes, then we can use `upsert` https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert